nexus/src/app/sagas/volume_delete.rs

```diff
@@ -44,9 +45,10 @@ use uuid::Uuid;
 
 #[derive(Debug, Deserialize, Serialize)]
 pub struct Params {
+    pub serialized_authn: authn::saga::Serialized,
+    pub project_id: Uuid,
```

Comment:
Just noting that this implicitly depends on Volumes not moving between Projects.  I've generally been trying to avoid that because I imagine we will eventually support that...but it seems like avoiding this dependency here is probably not worth it.